views/doc.list.html.twig

```diff
@@ -3,36 +3,61 @@
 {% import _self as macros %}
 
 {% block content %}
-    <div id="searchbar" class="clearfix">
-        <input id="docsearch" type="text" placeholder="Search documentation">
-    </div>
-
-    <h1>Book</h1>
-    <ul>
-    {% for filename, data in book %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>Articles</h1>
-    <ul>
-    {% for filename, data in articles %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>FAQs</h1>
-    <ul>
-    {% for filename, data in faqs %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-
-    <h1>API Docs</h1>
-    <p>If you need to work with composer as a library you may also want to check out the <a href="/apidoc/master/index.html">API Docs</a></p>
+    <article role="article">
```

Comment:
`role="article"` is useless. This is already the built-in semantic of `<article>`

Reply:
Oh, I see. Didn't know it was redundant to use it like that. I'll update everything! Thanks for pointing this out.